### ISSUE TITLE
Pauta do Comitê de Políticas Digitais no CONTEC

### ISSUE THREAD

**Issue by uira** [EscolaDeSaudePublica/TransformacaoDigital#1]
## #ProgramaDeTransformaçãoDigital
- Estruturar a Política de Bloqueios da Internet
- Contrato de Internet (Prioridade)
- Suite de Escritório (email, agenda e documentos)
- Registro de Demandas e Protocolo Único
- Gestão "Eletrônica" de Documentos (e processos)
- Gestão de Projetos e Atividades (SCRUM)

## Inovação
- Integração de Serviços e Sistemas ESP
- Identidade Digital > Provedor Público
- Mapa Digital > Mapa da Saúde
- Susi > Assistente Pessoal da Saúde

**Comment by Marceloalcantara** [comment]
Precisamos tb de uma ferramenta de gestão com funcionalidades “similares”’ao que o Trello oferece

**Comment by uira** [comment]
@Marceloalcantara Sim. Essa demanda está contemplada no item "Gestão de Projetos e Atividades"... 

A proposta, para todos os itens da pauta, é já definir uma solução ou apontar as diretrizes para estudo e apresentação das propostas de solução na próxima reunião.
